artison.py

```diff
@@ -82,8 +90,9 @@ def _to_json(self, data):
     def write_packer(self, filepath, builder='ec2'):
         with open(filepath, 'w') as f:
             f.write(self._to_json(self._compile_packer(builder)))
+        self._validate_packer_file(filepath)   
 
 
 if __name__ == "__main__":
-    d1 = artisan(None, 'conf/artisan.yml')
```

Comment:
This keeps reading from the yaml file relative to the project, does it have to change?